Packs/FeedMicrosoftIntune/TestPlaybooks/playbook-FeedMicrosoftIntune_Test.yml

```
@@ -0,0 +1,95 @@
+id: playbook-FeedMicrosoftIntune_Test
+name: playbook-FeedMicrosoftIntune_Test
+version: -1
+fromversion: 4.5.0
+starttaskid: '0'
+tasks:
+  '0':
+    id: '0'
+    taskid: '0'
+    type: start
+    task:
+      id: '0'
+      version: -1
+      name: ''
+      iscommand: false
+      brand: ''
+      description: ''
+    nexttasks:
+      '#none#':
+      - '1'
+    separatecontext: false
+    view: '{"position": {"x": 50, "y": 50}}'
+    note: false
+    timertriggers: []
+    ignoreworker: false
+  '1':
+    id: 1
+    taskid: '1'
+    type: regular
+    task:
+      id: '1'
+      version: -1
+      name: DeleteContext
+      description: ''
+      script: DeleteContext
+      type: regular
+      iscommand: true
+      brand: ''
+    nexttasks:
+      '#none#':
+      - '2'
+    scriptarguments:
+      all:
+        simple: yes
+    separatecontext: false
+    view: '{"position": {"x": 50, "y": 200}}'
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+  '2':
+    id: 2
+    taskid: '2'
+    type: regular
+    task:
+      id: '2'
+      version: -1
+      name: intune-get-indicators
+      description: ''
+      script: '|||intune-get-indicators'
+      type: regular
+      iscommand: true
+      brand: ''
+    nexttasks:
+      '#none#':
+      - '3'
+    scriptarguments: {}
```

In the test playbook, we need to verify that the outputs of this command are what we expect but since the `intune-get-indicators` command doesn't return output context it's a bit more complicated, need to extend the context in this task and then add another task that verifies the extended context. So here it would look something like the following,
```suggestion
    scriptarguments: 
      extend-context:
        simple: first_object=.[0]
```
To see an example, you can look at the test playbook in the Recorded Future Feed [test playbook](https://github.com/demisto/content/blob/master/Packs/FeedRecordedFuture/TestPlaybooks/playbook-RecordedFutureFeed.yml) or any other test playbook for our feed integrations.